.github/workflows/build.yml

```diff
@@ -17,4 +17,4 @@ jobs:
           go-version: 1.18
 
       - name: Test
-        run: make test MONGODB_STARTUP_TIMEOUT=8 GOTEST_FLAGS="-v -count=1 -race"
+        run: make test-integration MONGODB_STARTUP_TIMEOUT=8 GOTEST_FLAGS="-v -count=1 -race"
```

Comment:
We should run both, unit tests and integration tests, when testing builds.

Reply:
But what is the point to split unit tests from integration and acceptance tests?

Reply:
It might be a matter of preference, but at least I run only the unit tests while working on certain parts of the code. Only when I'm fully done I run the integration tests (just in case).

Reply:
Got it 👍 
Done.